screenfetch-dev

```diff
@@ -991,7 +991,7 @@ detectcpu () {
 			cpu="Unknown"
 		fi
 	elif [[ "$distro" == "FreeBSD" || "$distro" == "DragonflyBSD" ]]; then
-		cpu=$(sysctl -n hw.model)
+		cpu=$(dmesg | grep CPU: | head -n 1 | sed -r "s/CPU: //" | sed -e 's/([^()]*)//g')
```

Comment:
Ah! Yes, apologies. That was an update for compatibility  with Raspberry Pis and other ARM devices running FreeBSD.

```
[mscs@ate ~]$ sysctl -n hw.model
sysctl: unknown oid 'hw.model': No such file or directory
[mscs@ate ~]$ dmesg | grep CPU: | head -n 1 | sed -r "s/CPU: //" | sed -e 's/([^()]*)//g'
ARM1176JZ-S rev 7 


[mscs@nyx ~]$ dmesg | grep CPU: | head -n 1 | sed -r "s/CPU: //" | sed -e 's/([^()]*)//g'
AMD FX-8320 Eight-Core Processor            
[mscs@nyx ~]$ sysctl -n hw.model
AMD FX(tm)-8320 Eight-Core Processor           
```

a test on an Intel machine running FreeBSD might be a good idea.  I'll see if I can run one tomorrow.